site-src/geps/gep-1709.md

```diff
@@ -0,0 +1,44 @@
+# GEP-1709: Conformance Profiles
```

Comment:
Needs a link in mkdocs.yml since we haven't automated this yet.

Reply:
Will follow up, thank you :+1: